Data/fragmentdata.txt

```diff
@@ -83,7 +83,7 @@
 	{
 		"Id":10,
 		"Title":"At a quarry in Yagouchi",
-		"Description":"Help from the queen carrier is required.",
+		"Description":"Willingness to help from the queen carrier is required.",
```

Comment:
```suggestion
		"Description":"Cooperation from the queen carrier is required.",
```

Reply:
For reference, the phrasing used in the fragment is "Rika is willing to help"

https://github.com/07th-mod/matsuribayashi/blob/89ed4b2dd1c425ebca24889979de4a3eba3f31de/Update/_kakera07.txt#L829

Reply:
Actually I linked the wrong fragment, the actual wording in fragment 12 is "I'm willing to participate"

https://github.com/07th-mod/matsuribayashi/blob/714015600b14fb7270b5b9917b7e9a4a867bf098/Update/_kakera12.txt#L878